File: Fundamentals_of_Software_Design/1_Basic_skills/Вклад в банке/main.c
#include <stdio.h>
#include <math.h>

void calculate_deposit(double amount, int months, double annual_rate) {
    double monthly_rate = annual_rate / 12 / 100;
    double simple_interest = amount;
    double compound_interest = amount;

    printf("%-6s %-15s %-15s\n", "Месяц", "Простые %", "Сложные %");
    printf("----------------------------------------\n");

    for (int month = 1; month <= months; month++) {
        simple_interest += amount * (annual_rate / 100) / 12;
        compound_interest *= (1 + monthly_rate);
        printf("%-6d %-15.2f %-15.2f\n", month, simple_interest, compound_interest);
    }
}

int main() {
    double amount, annual_rate;
    int months;

    printf("Введите сумму вклада: ");
    scanf("%lf", &amount);
    printf("Введите срок вклада (в месяцах): ");
    scanf("%d", &months);
    printf("Введите годовой процент: ");
    scanf("%lf", &annual_rate);

    calculate_deposit(amount, months, annual_rate);

    return 0;
}

File: Fundamentals_of_Software_Design/1_Basic_skills/Вклад в банке/main.py
def calculate_deposit(amount, months, annual_rate):
    monthly_rate = annual_rate / 12 / 100
    simple_interest = amount
    compound_interest = amount

    print(f"{'Месяц':<6} {'Простые %':<15} {'Сложные %':<15}")
    print("-" * 40)

    for month in range(1, months + 1):
        simple_interest += amount * (annual_rate / 100) / 12
        compound_interest *= (1 + monthly_rate)
        print(f"{month:<6} {simple_interest:<15.2f} {compound_interest:<15.2f}")

amount = float(input("Введите сумму вклада: "))
months = int(input("Введите срок вклада (в месяцах): "))
annual_rate = float(input("Введите годовой процент: "))

calculate_deposit(amount, months, annual_rate)
